src/rules/no-import-module-exports.js

```diff
@@ -0,0 +1,72 @@
+import minimatch from 'minimatch'
+import path from 'path'
+import readPkgUp from 'read-pkg-up'
+
+function getEntryPoint(context) {
+  try {
```

Comment:
hmm...I'm not sure I follow.
From what I understand `require.resolve` gives you the absolute path to a given module.
So if I wanted the path to `minimatch`'s package.json I could do:
```js
require.resolve('minimatch/package.json')
```
But I can't use it to get an absolute path to the current module's package.json...Or can I? 

By the way, I borrowed this approach from [no-extraneous-dependencies](https://github.com/benmosher/eslint-plugin-import/blob/master/src/rules/no-extraneous-dependencies.js). If we come up with a better solution, it might not be a bad idea to implement it there as well.